src/components/Item/Item.js

```diff
@@ -145,6 +146,8 @@ export const Item = ({
 
   const avatar = <img {...{ src: items[id] }} alt={name} />
 
+  adjustedValue *= getSalePriceMultiplier(completedAchievements)
```

Comment:
Rather than mutating `adjustedValue` here, I think we're going to need to add an additional number to represent `adjustedValue * getSalePriceMultiplier(completedAchievements)`. The reason for this is that `adjustedValue`, at least for non-Shop items, specifically defines the value of an item adjusted for current market values. Modifying it here will break the calculations at these points deeper in the component:

https://github.com/jeremyckahn/farmhand/blob/26d249bac2678abda5e3417d951ff69224632d92/src/components/Item/Item.js#L204-L206

https://github.com/jeremyckahn/farmhand/blob/26d249bac2678abda5e3417d951ff69224632d92/src/components/Item/Item.js#L233-L235

This is a bit confusing because #73 makes the semantics of this variable a bit unclear. It may make sense to rename `adjustedValue` to something like `marketAdjustedValue`, since the current name is too vague now. I think "adjustedValue" is good for representing "the final number that is presented to the user," in case you need some help with the variable naming here.

Reply:
i see now, thanks for pointing that out. i think it actually makes the most sense to add a `sellPrice` variable and only adjust the "Sell price" field. i would like to see `adjustedValue` updated as you said, but there are many other similar variables that i would also want to update like `previousDatAdjustedValue` and `historicalValueAdjustments` so I think it's best to leave those all for now. how does that sound?

Reply:
Oh yeah, those variables are functionally affected by this feature at all, and changing their name is actually quite a bit more involved because they are part of the persisted game data. They're named slightly more vaguely than I'd like right now, but it's not worth changing them in this PR.